docs/integration/ecs-logs.md

```diff
@@ -0,0 +1,179 @@
+title: Elastic Container Service (ECS) Logs Integration
+description: Whether you're using EC2 or Fargate, forward all container logs to Sematext Cloud. Get insight into your whole ECS Cluster.
+
+By configuring either Logagent for EC2 or AWS Firelens for Fargate, you can forward all your container logs to Sematext Logs and get insight into your whole Elastic Container Service (ECS) cluster in one place!
+
+## ECS Logs Quick Start
+
+The ECS Logs Integration collects logs from ECS Tasks and Services running in:
+
+- EC2 Container Instances
+- Fargate 
+
+To collect logs from EKS, see the [Kubernetes integration](./kubernetes/#shipping-kubernetes-logs-to-sematext).
+
+## Forwarding ECS Fargate logs to Sematext
+
+There are two main ways you can forward logs from containers running in Fargate to Sematext. They rely on two different log drivers.
+
+- [AWS Firelens](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html) - `awsfirelens`
+- [AWS Logs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html) - `awslogs`
+
+We suggest you use AWS Firelens to avoid additional CloudWatch costs.
+
+### AWS Firelens
+With Firelens you can route logs to another AWS service, like Firehose, or use Fluentd or Fluent Bit. AWS provides the image for Fluentd / Fluent Bit. You need to configure the output module.
+
+#### 1. Enable Firelens
+In the ECS Task Definition, check a checkbox called Enable FireLens integration. Choose Fluent Bit and AWS will populate the image name for you.
+
+![aws-ecs-firelens-1](../images/integrations/aws-ecs-firelens-1.png)
+
+An additional container called `log_router` will show up.
```

Comment:
Where will it show up?  UI?  Sounds like AWS will launch it?  If so, maybe:
AWS will launch an additional container called `log_router`.

Reply:
Yes, in the UI. So this is a better way of saying it:

> AWS will add an additional container called `log_router` to the list of containers in your Task Definition

Reply:
OK